src/main/java/core/basesyntax/service/impl/OperationProcessingImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.OperationProcessing;
+import core.basesyntax.service.OperationProcessingStrategy;
+import java.util.List;
+
+public class OperationProcessingImpl implements OperationProcessing {
```

Comment:
```suggestion
public class OperationProcessingServiceImpl implements OperationProcessingService {
```
operation prosessing is a process that the Service does, lets add Service to name